Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -269,20 +269,23 @@ class BrowserViewController: UIViewController {
 
         appMenuBadgeUpdate()
 
-        if showTopTabs, topTabsViewController == nil {
+        if showTopTabs, self.topTabsViewController == nil {
```

Comment:
Why do we need to add `self.` in those places?

Reply:
I thought that it would help to avoid confusion based on the local `topTabsViewController`. But you are right, it's probably unnecessary.